From 9997ff71a7ffbdfc00adabea882d79c25e1abe24 Mon Sep 17 00:00:00 2001
From: Daniel Wang <wonderfly@google.com>
Date: Fri, 14 Feb 2020 14:39:06 -0800
Subject: [PATCH] systemd: Make default.target point to multi-user.target

The upstream default, graphical.target, doesn't make sense for us.
---
 units/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/units/meson.build b/units/meson.build
index e4ac6ced64..6f3f35619f 100644
--- a/units/meson.build
+++ b/units/meson.build
@@ -17,7 +17,7 @@ units = [
          'multi-user.target.wants/'],
         ['getty-pre.target',                    ''],
         ['graphical.target',                    '',
-         'runlevel5.target default.target'],
+         'runlevel5.target'],
         ['halt.target',                         ''],
         ['hibernate.target',                    'ENABLE_HIBERNATE'],
         ['hybrid-sleep.target',                 'ENABLE_HIBERNATE'],
@@ -36,7 +36,7 @@ units = [
         ['machines.target',                     'ENABLE_MACHINED',
          join_paths(pkgsysconfdir, 'system/multi-user.target.wants/')],
         ['multi-user.target',                   '',
-         'runlevel2.target runlevel3.target runlevel4.target'],
+         'runlevel2.target runlevel3.target runlevel4.target default.target'],
         ['network-online.target',               ''],
         ['network-pre.target',                  ''],
         ['network.target',                      ''],
-- 
2.25.0.265.gbab2e86ba0-goog

